streams/src/main/java/org/apache/kafka/streams/kstream/internals/KTableImpl.java

```diff
@@ -243,4 +247,27 @@ public void to(String topic, Serializer<K> keySerializer, Serializer<V> valSeria
         return new KTableImpl<>(topology, joinMergeName, joinThis, allSourceNodes);
     }
 
+    @Override
+    public <K1, V1, V2> KTable<K1, V2> aggregate(AggregatorSupplier<K1, V1, V2> aggregatorSupplier, KeyValueMapper<K, V, KeyValue<K1, V1>> selector, String name) {
+        // TODO
+        return null;
+    }
+
+    @Override
+    public <K1, V1> KTable<K1, Long> sum(KeyValueMapper<K, V, KeyValue<K1, V1>> selector, ValueMapper<V1, Long> valueMapper, String name) {
+
+        return this.aggregate(new SumSupplier<>(valueMapper), selector, name);
+    }
+
+    @Override
+    public <K1> KTable<K1, Long> count(KeyValueMapper<K, V, KeyValue<K1, Long>> selector, String name) {
```

Comment:
Shouldn't the first argument be `KeyValueMapper<K, V, K1> selector`? We only care the aggregation key.